packages/integration-test-app/windows/integrationtest/integrationtest.vcxproj

```diff
@@ -14,6 +13,11 @@
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
   </PropertyGroup>
+  <PropertyGroup Label="NuGet">
+    <ResolveNuGetPackages>false</ResolveNuGetPackages>
+    <!-- https://github.com/NuGet/Home/issues/10511#issuecomment-778400668 -->
+    <AssetTargetFallback>$(AssetTargetFallback);uap10.0.16299</AssetTargetFallback>
+  </PropertyGroup>
```

Comment:
Where are they? I thought these projects didn't include, say, `React.Cpp.props`.

Reply:
See `vnext\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.props`.

The "external" property sheets are kept local to the `react-native-winows` NPM package, instead of the user's project. This means that we can control the props their projects will see without having to change the build logic for their code.